neuralhydrology/modelzoo/transformer.py

```diff
@@ -34,10 +34,12 @@ class Transformer(BaseModel):
     cfg : Config
         The run configuration.
     """
-
     def __init__(self, cfg: Dict):
         super(Transformer, self).__init__(cfg=cfg)
 
+        # specify submodules of the model that can later be used for finetuning. Names must match class attributes
+        self.module_parts = ['embedding', 'encoder', 'head']
```

Comment:
```suggestion
        self.module_parts = ['embedding_net', 'encoder', 'head']
```
strings must match the names of the class members